runtime/lua/vim/_defaults.lua

```diff
@@ -23,7 +23,11 @@ do
 
   vim.api.nvim_create_user_command('EditQuery', function(cmd)
     vim.treesitter.query.edit(cmd.fargs[1])
-  end, { desc = 'Edit treesitter query', nargs = '?' })
+  end, {
+    desc = 'Edit treesitter query',
+    nargs = '?',
+    complete = vim.treesitter.language._complete,
```

Comment:
perf nit: vim.ts is [lazy loaded](https://github.com/neovim/neovim/blob/7cd5356a6f89a46d83bbba9b7f6496b67f054629/runtime/lua/vim/treesitter.lua#L6) so let's wrap this to avoid eager-loading it:
```suggestion
    complete = function(...) return vim.treesitter.language._complete(...) end,
```

Reply:
works like a charm 👨‍🍳